build.gradle

```diff
@@ -212,6 +212,7 @@ configure(rootProject) { project ->
 	compile "io.netty:netty-handler-proxy:${nettyVersion}"
 	compile "io.netty:netty-codec-http:${nettyVersion}"
 	compile "io.netty:netty-transport-native-epoll:${nettyVersion}"
+	compile "io.netty:netty-transport-native-kqueue:${nettyVersion}"
```

Comment:
will need to make that provided like slf4j in reactor-core.

Reply:
I made it optional